[BUG] Failed to download Chromium on the bots

https://github.com/microsoft/playwright/pull/5229/checks?check_run_id=1803708222

Downloading browsers...
Done. Use "npm run watch" to compile.
Failed to install browsers, caused by
Error: Failed to download chromium, caused by
Error: connect ETIMEDOUT 13.107.246.13:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1107:14)
    at browserFetcher.downloadBrowserWithProgressBar.catch.e (/Users/runner/work/playwright/playwright/lib/install/installer.js:106:19)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! playwright-internal@1.9.0-next prepare: `node install-from-github.js`
npm ERR! Exit status 1

--- a/src/install/browserFetcher.ts
+++ b/src/install/browserFetcher.ts
@@ -137,7 +137,8 @@ export async function downloadBrowserWithProgressBar(browsersPath: string, brows
       const {error} = await downloadFile(url, zipPath, progress);
       if (!error)
         break;
-      if (attempt < N && error && typeof error === 'object' && typeof error.message === 'string' && error.message.includes('ECONNRESET')) {
+      const errorMessage = typeof error === 'object' && typeof error.message === 'string' ? error.message : '';
+      if (attempt < N && (errorMessage.includes('ECONNRESET') || errorMessage.includes('ETIMEDOUT'))) {
         // Maximum delay is 3rd retry: 1337.5ms
         const millis = (Math.random() * 200) + (250 * Math.pow(1.5, attempt));
         await new Promise(c => setTimeout(c, millis));

